### PR TITLE
Added 'language' field to all tracking events

### PR DESCRIPTION
**Pull request by radujipa**
Related to https://trello.com/c/8HLyQFpt/87-add-tracking-event-that-lets-us-know-a-kit-is-in-argentina

@tombettany @skarbat

### CODE REVIEW

**Comment by radujipa** [comment]
[Add tracking event that lets us know a kit is in argentina](https://trello.com/c/8HLyQFpt/87-add-tracking-event-that-lets-us-know-a-kit-is-in-argentina)

**Comment by skarbat** [comment]
Good idea @radujipa . Perhaps a try/catch to protect against exceptions on pulling `LANG` ?
I've seen an empty `LC_CTYPE` one the devel image.

File: kano_profile/tracker.py [hunk 1]
```diff
@@ -107,6 +107,7 @@ def generate_tracker_token():
 OS_VERSION = str(read_file_contents('/etc/kanux_version'))
 CPU_ID = str(get_cpu_id())
 TOKEN = load_token()
+LANGUAGE = (os.getenv('LANG') or '').split('.', 1)[0]
```
Comment (radujipa) [inline kano_profile/tracker.py]: @skarbat I have also somehow encountered `None` values for getting the `LANG` value, hence:

The following can throw an exception if the key in not in the dict:
```
os.environ['LANG']`
```

However, the following doesn't throw an exception, but may return `None` instead:
```
os.getenv('LANG')
```

Comment (skarbat) [inline kano_profile/tracker.py]: Oh yes, very true! sorry about confusion. 👍

**Pull request merged**